crates/red_knot_python_semantic/resources/mdtest/unreachable.md

```diff
@@ -218,26 +255,24 @@ def f():
 
 In the example below, since we use `x` in the `inner` function, we use the "public" type of `x`,
 which currently refers to the end-of-scope type of `x`. Since the end of the `outer` scope is
-unreachable, we treat `x` as if it was not defined. This behavior can certainly be improved.
```

Comment:
Nothing changed in this test, but this explanation was outdated. And the `return x  # …` below as confusing.